### ISSUE TITLE
Create new view in Jupyterlab

### ISSUE THREAD

**Issue by hadim** [holoviz/panel#483]
Is the "Create New View from Output" feature in Jupyterlab supposed to work? Because it does not seem to be working at the moment.

Steps to reproduce:

- Fire Jupyterlab and open a notebook.
- Add the following code:

```python 
import panel as pn
pn.extension()

gspec = pn.GridSpec(width=800, height=600)

gspec[:,   0  ] = pn.Spacer(background='red',    margin=0)
gspec[0,   1:3] = pn.Spacer(background='green',  margin=0)
gspec[1,   2:4] = pn.Spacer(background='orange', margin=0)
gspec[2,   1:4] = pn.Spacer(background='blue',   margin=0)
gspec[0:1, 3:4] = pn.Spacer(background='purple', margin=0)

gspec
```

- Execute. You should the panel displaying.
- Right click on the panel and select "Create New View from Output".
- A new Jupyter lab window open but it stays empty.
- While in the notebook, a second identical output cell appeared below the previous one.

![image](https://user-images.githubusercontent.com/528003/59565330-1e1bc380-9020-11e9-81fd-21e7962f2961.png)

**Comment by hadim** [comment]
(also let me know if I should post on the Jupyter lab repo instead)

**Comment by hadim** [comment]
So according to https://github.com/jupyterlab/jupyterlab/issues/6594, this feature should works.

I am not very familiar with the internal logic of Panel but could that be related to the `pn.extension()` we run at the beginning that inject some JS/CSS magic that is not injected (I guess) in the new output view?

**Comment by xavArtley** [comment]
Just for the record this issue is known from a long time ago and not solved for the moment.
https://github.com/pyviz/pyviz_comms/issues/3

**Comment by NumesSanguis** [comment]
Also found the same problem in Panel. Added my encounter in @xavArtley 's linked issue: https://github.com/holoviz/pyviz_comms/issues/3#issuecomment-559336440

**Comment by philippjfr** [comment]
This was resolved in 0.14.
